package.json

```diff
@@ -5,8 +5,8 @@
   "main": "webpack.config.js",
   "scripts": {
     "check-licenses": "wp-scripts check-licenses --prod --gpl2",
-    "build": "wp-scripts build --webpack-no-externals",
-    "dev": "wp-scripts start --webpack-no-externals"
```

Comment:
This means there is no `index.asset.php` file, and the `.js` file isn't enqueued.